mtp_api/apps/security/management/commands/update_security_profiles.py

```diff
@@ -0,0 +1,107 @@
+from django.db.models import Max
+from django.core.management import BaseCommand
+
+from credit.models import Credit
+from security.models import (
+    SenderProfile, BankTransferSenderDetails, DebitCardSenderDetails,
+    PrisonerProfile, SecurityDataUpdate, CardholderName
+)
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        try:
+            last_updated_at = SecurityDataUpdate.objects.latest().updated_at
+            new_credits = Credit.objects.filter(received_at__gt=last_updated_at)
+        except SecurityDataUpdate.DoesNotExist:
+            new_credits = Credit.objects.all()
+
+        for credit in new_credits:
+            self.create_or_update_profiles(credit)
+
+        if len(new_credits):
```

Comment:
since you never access `new_credits` contents directly, seems to me that `new_credits.exists()` is more appropriate than loading all of them